voyager_archive/views_utils.py

```diff
@@ -1,14 +1,36 @@
 from django.shortcuts import get_object_or_404
+from pymarc import Record
 from .models import AuthRecord, BibRecord, MfhdRecord
 
 
 def get_auth_record(auth_id: int) -> AuthRecord:
-    return get_object_or_404(AuthRecord, auth_id=auth_id)
```

Comment:
Exception handling is important in these, as users may search for records which do not exist.
https://docs.djangoproject.com/en/4.1/topics/http/shortcuts/#get-object-or-404 was convenient for the QAD implementation I did, but may need to be replaced with more explicit handling as shown in an example at that link.